### PR TITLE
Add new words to triadespeech

### PR DESCRIPTION
**Pull request by NiklasJan**

### CODE REVIEW

File: src/main/java/de/fuzzlemann/ucutils/utils/faction/badfaction/speech/TriadeSpeechModifier.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    public static final List<Map.Entry<String, String>> REPLACE_IGNORE_CASE = Lists.newArrayList(
+
+            Maps.immutableEntry("Nia Hao", "Halllo"),
+            Maps.immutableEntry("Shi De", "Ja"),
+            Maps.immutableEntry("Bu Shi", "Nein"),
+            Maps.immutableEntry("Zai Jian", "Tschüss"),
+            Maps.immutableEntry("Xie Xie", "Danke")
+
+    );
+
```
Comment (paulzhng) [inline src/main/java/de/fuzzlemann/ucutils/utils/faction/badfaction/speech/TriadeSpeechModifier.java:24]: remove the empty lines besides one separating `);` and `@Override`

File: src/main/java/de/fuzzlemann/ucutils/utils/faction/badfaction/speech/TriadeSpeechModifier.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import java.util.Map;
+import java.util.StringJoiner;
+
+/**
+ * @author Fuzzlemann
+ */
+public class TriadeSpeechModifier implements SpeechModifier {
+
+    public static final List<Map.Entry<String, String>> REPLACE_IGNORE_CASE = Lists.newArrayList(
+
```
Comment (paulzhng) [inline src/main/java/de/fuzzlemann/ucutils/utils/faction/badfaction/speech/TriadeSpeechModifier.java:16]: ^ this line too

File: src/main/java/de/fuzzlemann/ucutils/commands/faction/CallReinforcementCommand.java [hunk 3]
```diff
@@ -217,6 +217,7 @@ public boolean onCommand(UPlayer p,
         MEDIC("-m", "d", "Medic benötigt!"),
         CORPSE_GUARDING("-lb", "d", "Leichenbewachung!"),
         DRUG_REMOVAL("-da", "d", "Drogenabnahme!"),
+        CONTRACT("-ct", "f", "Contract!"),
```
Comment (paulzhng) [inline src/main/java/de/fuzzlemann/ucutils/commands/faction/CallReinforcementCommand.java:220]: Please make a separate PR for that

**Review by paulzhng — CHANGES_REQUESTED** [review]

**Comment by NiklasJan** [comment]
#40
